### PR TITLE
Feature/button

### PR DESCRIPTION
**Pull request by andresV74**
Button types

This component renders all the above type of buttons including it rollover state.

I'm still trying to make it look right in Storybooks.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,12 +12,13 @@
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-icons": "^3.11.0",
     "react-scripts": "^3.4.3",
     "styled-components": "^5.1.1"
   },
   "devDependencies": {
     "@babel/core": "^7.11.1",
+    "@iconify/icons-bx": "^1.0.3",
```
Comment (wolftrax5) [inline package.json:20]: so the icon library will be iconify ?

File: src/components/Button/Button.stories.js [hunk 2]
```diff
@@ -2,13 +2,12 @@ import React from 'react';
 
 import { action } from '@storybook/addon-actions';
 // REF https://www.learnstorybook.com/intro-to-storybook/react/en/simple-component/
-import Button from '.';
+import Button from './index';
```
Comment (wolftrax5) [inline src/components/Button/Button.stories.js:5]: Why add ./index ? on an import js the default is the index

File: package.json (hunk 1 above)
Comment (andresV74) [reply package.json:20]: Yes, there is an issue resizing the icon with react-icons

File: src/components/Button/index.js [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
+import { InlineIcon } from '@iconify/react';
+import { Buttons } from './styles';
 
-const Button = ({ children, buttonType, onClick }) => (
-  <ButtonS buttonType={buttonType} onClick={onClick}>
+const Button = ({ children, buttonType, onClick, icon, iconSmall }) => (
+  <Buttons buttonType={buttonType} onClick={onClick}>
     {children}
-  </ButtonS>
+    {icon ? (
```
Comment (wolftrax5) [inline src/components/Button/index.js:9]: Why is this logical? on both case arre render the same `<InlineIcon icon={iconSmall} />` or `<InlineIcon icon={iconSmall} />` what is the diference on both properties ?
`{icon ? ( <InlineIcon icon={icon} />) : (''")}` omit the iconSmall validation will have the same resutls no ?

File: src/components/Button/Button.stories.js (hunk 2 above)
Comment (andresV74) [reply src/components/Button/Button.stories.js:5]: yes, you're right, I will fix it asap

File: src/components/Button/index.js (hunk 3 above)
Comment (andresV74) [reply src/components/Button/index.js:9]: you're right, I made that logic when I was dealing with react-icons and when I switched to iconify I forgot to fix it

**Pull request merged**